lib/test-utils/src/Test/QuickCheck/Extra.hs

```diff
@@ -208,6 +213,61 @@ shrinkInterleaved (a, shrinkA) (b, shrinkB) = interleave
 -- Evaluating shrinkers
 --------------------------------------------------------------------------------
 
+-- | Computes the entire shrink space of a given value and shrinking function.
```

Comment:
It might be worth including a note to say that the shrink space of a given value and shrinking function is not necessarily equal to the entire value space of a type.

For example if the shrinking function for `Int` never chooses the value `7`, this function will also never produce the value `7`.

That became obvious to me after reading the implementation of this function and thinking about the type signature, but I don't think it's obvious on first glance.

Reply:
Hi @sevanspowell 

I've tried to clarify the comment for `shrinkSpace` a bit, and give some more worked examples.

See: ce310edacc949f1c751a4e7fcf04d6e1007221c7